app/main/lib/text_similarity.py

```diff
@@ -142,9 +142,13 @@ def strip_vectors(results):
 
 def restrict_results(results, search_params, model_key):
     out_results = []
-    if search_params.get("min_es_score") and model_key == "elasticsearch":
+    try:
+        min_es_score = float(search_params.get("min_es_score"))
+    except:
```

Comment:
is this to catch number format exception?   if so, can it catch that specific exception, log the error and re-raise so that the calling server process will send the error message back?